examples/custom-databases/mongoose.js

```diff
@@ -0,0 +1,116 @@
+const Discord = require('discord.js'),
+    client = new Discord.Client(),
+    settings = {
+        prefix: 'g!',
+        token: 'Your Discord Bot Token',
+    };
+
+// Connect to database
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/giveaways', { useFindAndModify: false });
+const db = mongoose.connection;
+
+// Check the connection
+db.on('error', console.error.bind(console, 'Connection error:'));
+db.once('open', () => {
+    console.log('Connected to MongoDB.');
+});
+
+// Create the schema for giveaways
+const giveawaySchema = new mongoose.Schema({
+    messageID: String,
+    channelID: String,
+    guildID: String,
+    startAt: Number,
+    endAt: Number,
+    ended: Boolean,
+    winnerCount: Number,
+    winners: [],
```

Comment:
```suggestion
```
I think you meant `winnerIDs` but you already added a little further down below

Reply:
Ah, minor mistake!

Reply:
Hey, my giveaways don't seem to be updating again once I restart the Bot. Is there any way to make the Bot regularly make sure all of the Giveaways are updated? If so, how?

Reply:
> Hey, my giveaways don't seem to be updating again once I restart the Bot. Is there any way to make the Bot regularly make sure all of the Giveaways are updated? If so, how?

First, don't ask that in a code suggestion... Second, have you tried using only the connection code from the example? Because you had some custom options.

Reply:
I will try with just yours.

Reply:
Still having the same issue sadly.

Reply:
> Still having the same issue sadly.

Could you PLEASE start commenting on the pr itself and not in this already resolved code change? THANKS!